### ISSUE TITLE
CUDA_ERROR_ILLEGAL_ADDRESS error

### ISSUE THREAD

**Issue by archer0646** [trexminer/T-Rex#1162]
Does anyone know how to solve this error? I did reduce overclock, increased virtual memory, updated drivers. I have mined with other software with no issues.
![Capture](https://user-images.githubusercontent.com/101531343/158083485-dab689d2-350c-44b7-a87d-e0506fa89137.PNG)

**Comment by lcssen** [comment]
Graphics driver back to NVIDIA 472.12 to test!

**Comment by archer0646** [comment]
Switching to ETH+ALPH works fine. Seems like this issue is only occurring with ETH+ERGO

**Comment by allenultra** [comment]
Tried this before, my own experience, its something to do with the t-rex version. Certain version, it will work just fine ....

**Comment by storopoli** [comment]
I can also confirm this issue in dual mining ETH+ERGO GeForce RTX 3700Ti

**Comment by voxitme** [comment]
Same here, but exclusively when mining under windows 10: no such issue under latest hiveOS.

**Comment by Mardreck** [comment]
Same here, 3070 ti, Windows 11.
I just posted

**Comment by coryb13** [comment]
I'm running into this same thing, and I noticed an interesting pattern. Only the 3070 Ti is affected. Other cards will run fine, even if they're on the same computer as a 3070 Ti, as long as the 3070 Ti is excluded.

All the following work if you don't include the 3070 Ti in the bat file:
3060 Ti
3080
3080 Ti

I also tested 3 different Nvidia drivers and 3 different T-rex version, and I'm getting the same error on all of them.

**Comment by archer0646** [comment]
Error still occuring on version 0.25.9, `--dataset-mode 1` does not seem to fix it as suggested in thread #1173

**Comment by PrinceOfParallax** [comment]
> 

Confirmed does not work on any 8gb card I have.

**Comment by pachcolom** [comment]
I managed to get it to work. I had to downgrade T-Rex to v0.24.2.

It did not work with v0.24.4 and v0.24.5. I stopped testing T-Rex versions.

**Comment by voxitme** [comment]
> I managed to get it to work. I had to downgrade T-Rex to v0.24.2.
> 
> It did not work with v0.24.4 and v0.24.5. I stopped testing T-Rex versions.

Wow thanks my friend!
I did try down to 0.24.5 but at that point didn't expect anything from lower ones!
I can confirm this works on my side too, thanks for sharing!

**Comment by dittomining** [comment]
> I managed to get it to work. I had to downgrade T-Rex to v0.24.2.
> 
> It did not work with v0.24.4 and v0.24.5. I stopped testing T-Rex versions.

Have been dual mining ETH+ERGO for a few months on 2 3060ti's when I got this error last night on GPU1. Default clocks, driver update and increasing virtual memory had no effect, but switching to 0.24.2 worked as well! Not sure why this error suddenly appeared. Thank you!

**Comment by absolution1** [comment]
I have found something similar
 - 3060ti
 - Windows
 - dual mining eth+ergo for months 
 - Last night started seeing the illegal address error
 - Swapping back to 0.24.2 gets passed the error

What I am now finding though, is that the GPU keeps locking, and the LHR tune just keeps decreasing in value.  I DID update my drivers while debugging, so its a possible cause, I guess...

**Comment by voxitme** [comment]
> I have found something similar
> 
> * 3060ti
> * Windows
> * dual mining eth+ergo for months
> * Last night started seeing the illegal address error
> * Swapping back to 0.24.2 gets passed the error
> 
> What I am now finding though, is that the GPU keeps locking, and the LHR tune just keeps decreasing in value. I DID update my drivers while debugging, so its a possible cause, I guess...

Did you apply the same --lhr-tune parameter as with your previous (newer) version?

This fix might only be temporary, as ETH+ERGO will soon exceed 8GB VRAM.

**Comment by absolution1** [comment]
yeah I'm using the exact same BAT script.  The tune was set to 37.  I've looked at the dox and I think it mentioned starting from 30?  So, I've moved to that value.  Previously I was able to use a higher value though...

**Comment by voxitme** [comment]
That's quite weird, if you set it up to 37, I don't see why it should start from 30.
Are you calling the right .exe (see t-rex version at the launch in the terminal)?
-------- Original Message --------

**Comment by absolution1** [comment]
Sorry, I've confused things.
I was setting --lhr-tune 37, and indeed it was starting at 37, however it would pretty quickly start lowering the value during mining.
Checking the documentation online, there was a suggestion for starting at 30 instead.

Anyway, I think I'm going to move away from dual mining now, given that we're on the cusp of ERG+ETH being unminable on a 8GB card.  So, I'll start moving back towards solo eth instead.

Thank you for your help.

**Comment by mayronguevara** [comment]
> I have found something similar
> 
> * 3060ti
> * Windows
> * dual mining eth+ergo for months
> * Last night started seeing the illegal address error
> * Swapping back to 0.24.2 gets passed the error
> 
> What I am now finding though, is that the GPU keeps locking, and the LHR tune just keeps decreasing in value. I DID update my drivers while debugging, so its a possible cause, I guess...

that is correct, i have applied the same workaround using v. 0.24.2 is not showing the error as is mining dual.  the weird is sundenly stopped to work in the latest version, but not in that version.
